src/main/java/hellfirepvp/astralsorcery/common/entity/item/EntityItemHighlighted.java

```diff
@@ -33,9 +36,21 @@
 
     private static final DataParameter<Integer> DATA_COLOR = EntityDataManager.createKey(EntityItemHighlighted.class, DataSerializers.VARINT);
     private static final int NO_COLOR = 0xFF000000;
+    
+    private static Field skipPhysicRenderer;
+    
+    static {
+        try {
+            skipPhysicRenderer = ObfuscationReflectionHelper.findField(ItemEntity.class, "skipPhysicRenderer");
+        } catch (UnableToFindFieldException e) {}
+    }
 
     public EntityItemHighlighted(EntityType<? extends ItemEntity> type, World world) {
         super(type, world);
+        if(skipPhysicRenderer != null)
+            try {
+                skipPhysicRenderer.setBoolean(this, true);
```

Comment:
use ObfuscationHelper.setPrivateValue() instead; so we don't need to try/catch ourselves.

Reply:
Bad for performance, the field should be cached.

Reply:
Derp. my bad. Thought for a moment it took a Field not just the fieldname.